.reaction/pluginConfig.js

```diff
@@ -0,0 +1,5 @@
+const disabledPlugins = [];
```

Comment:
Can you add a comment above here explaining what goes in the array? At first glance I would have thought the plugin's name, but it looks like actually the directory name.

Also, should we gitignore this file? (after this initial commit)